src/Account/Controllers/AccountViewController.php

```diff
@@ -135,6 +143,42 @@ public function success(): Response
         return $this->render('accounts/register_success.html.twig');
     }
 
+    /**
+     * @param Request $request
+     * @return Response
+     */
+    public function login(Request $request): Response
+    {
+        $form = $this->createLoginForm();
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $data = $form->getData();
+
+            try {
+                $user = $this->userRepository->getByEmail(new Email($data['email']));
+
+                if ($user && $user->getPassword() && $user->getPassword()->verifies($data['password'])) {
```

Comment:
@stejes kan geen kwaad dat dit zo is aangezien `$user` null kan zijn en `getPassword` null kan teruggeven.